app/governance/proposal/page.tsx

```diff
@@ -162,37 +208,102 @@ export default function Page() {
           <Text opacity={0.4}>{proposal.description}</Text>
         </div>
       </div>
+      <div className={styles.proposalInfoContainer}>
+        <div className={styles.graphAndVoteContainer}>
+          {isActive && (
```

Comment:
Changed the Voting Option Boxes layout(as per the design) and containerization a bit to be consistent